source/android/adaptivecards/src/main/java/io/adaptivecards/renderer/layout/StretchableInputLayout.java

```diff
@@ -33,20 +44,38 @@ public void setInputView(View input)
     {
         addView(input);
         m_inputView = input;
+        if(m_inputView.getId() == View.NO_ID) {
+            m_inputView.setId(View.generateViewId());
+        }
+        m_label.setLabelFor(m_inputView.getId());
     }
 
     public View getErrorMessage()
     {
         return m_errorMessage;
     }
 
-    public void setErrorMessage(View errorMessage)
+    public void setErrorMessage(TextView errorMessage)
     {
         addView(errorMessage);
         m_errorMessage = errorMessage;
+        m_errorMessage.setImportantForAccessibility(IMPORTANT_FOR_ACCESSIBILITY_NO);
```

Comment:
>IMPORTANT_FOR_ACCESSIBILITY_NO [](start = 52, length = 30)

What does this signify? Does setting it to NO cause readers to skip reading the error message? Is that the behavior we want ?

Reply:
This is to avoid announcing the error multiple times.